.github/workflows/scripts/comments.mjs

```diff
@@ -1,14 +1,96 @@
+import { readFile } from 'node:fs/promises'
+
+/**
+ * Posts the content of multiple diffs in parallel on the given Github issue
+ *
+ * @param {GithubActionContext} githubActionContext
+ * @param {number} issueNumber
+ * @param {DiffComment[]} diffs
+ */
+export async function commentDiffs (
+  githubActionContext,
+  issueNumber,
+  diffs
+) {
+  // Run the comments in parallel to avoid that an early one failing prevents the others
+  // and use `allSettled` to avoid the promise rejecting on the first failure but wait
+  // for everything to complete
+  const results = await Promise.allSettled(
+    diffs.map((diff) =>
+      commentDiff(githubActionContext, issueNumber, diff)
+    )
+  )
+
+  const resultsWithErrors = results.filter(result => result.status === 'rejected')
```

Comment:
This is much nicer, thanks for passing `diffs` in 😊

For some reason TypeScript is typing `resultsWithErrors` incorrectly

```
PromiseSettledResult<void>[]
```

Might be similar to the [`.map()` inline typing](https://github.com/alphagov/govuk-frontend/blob/7fe712acc5f01da1a014f37b4c4a94ff17d0a6c9/shared/tasks/browser.mjs#L36) I had to do recently

I'll have a look tomorrow